packages/headless/doc-parser/src/interface-resolver.ts

```diff
@@ -158,6 +156,31 @@ function resolveMethodSignature(entry: ApiEntryPoint, m: ApiMethodSignature) {
   });
 }
 
+function buildEntityFromReturnType(m: ApiMethodSignature) {
+  const nodes = m.tsdocComment?.returnsBlock?.content.getChildNodes() || [];
+  const description = emitAsTsDoc((nodes as unknown) as readonly DocNode[]);
+
+  return buildEntity({
+    name: 'returnType',
+    type: m.returnTypeExcerpt.text,
+    isOptional: false,
+    comment: description,
```

Comment:
Ok, I see the reason for adding `string` to the comment interface. I would suggest instead creating a separate function `buildReturnTypeEntity` (essentially moving this function), similar to `buildParamEntity`, inside entity-builder. I think having such a function in a central location will be useful,